README.md

```diff
@@ -333,87 +333,142 @@ $ballerina run target/airline_backend_system.balx
+    registry:"ballerina.guides.io",
+    name:"airliine_reservation",
+    tag:"v1.0"
+}
 endpoint http:Listener airlineReservationEP {
     port: 9090
 };
 
-@Description { value: "Airline reservation service exposed via HTTP/1.1." }
+documentation { Airline reservation service exposed via HTTP/1.1. }
 @http:ServiceConfig {
     basePath: "/airline"
 }
 service<http:Service> AirlineReservation bind airlineReservationEP {
-``` 
+```
+
+##### flight_booking_backend.bal
+```ballerina
+import ballerinax/docker;
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mb;
+
+@docker:Config {
+    registry:"ballerina.guides.io",
+    name:"flight_booking_backend",
+    tag:"v1.0"
+}
+service<mb:Consumer> bookingListener bind { host: "<IP_ADDRESS_OF_BALLERINA_MESSAGE_BROKER_CONTAINER>",
+                                                                             port: 5672, queueName: "NewBookingsQueue" } {
 
-- `@docker:Config` annotation is used to provide the basic docker image configurations for the sample. `@docker:Expose {}` is used to expose the port. 
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        // Get the new message as the string
+        string messageText = check message.getTextMessageContent();
+        // Mock the processing of the message for a new reservation.
+        log:printInfo("[NEW BOOKING] Details : " + messageText);
+    }
+}
+
+service<mb:Consumer> cancellingListener bind { host: "<IP_ADDRESS_OF_BALLERINA_MESSAGE_BROKER_CONTAINER>",
+                                                                     port: 5672, queueName: "BookingCancellationQueue" } {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        // Get the new message as the string
+        string messageText = check message.getTextMessageContent();
+        // Mock the processing of the message to cancel a reservation
+        log:printInfo("[CANCEL BOOKING] : " + messageText);
+    }
+}
+```
+
+- `@docker:Config` annotation is used to provide the basic Docker image configurations for the sample. `@docker:Expose {}` is used to expose the port.
 
 - Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. It points to the service file that we developed above and it will create an executable binary out of that. 
-This will also create the corresponding docker image using the docker annotations that you have configured above. Navigate to `messaging-with-ballerina/guide` and run the following command.  
+This will also create the corresponding Docker image using the docker annotations that you have configured above. Navigate to `messaging-with-ballerina/guide` and run the following command.
+
 ```
-   $ ballerina build flight_booking_service
+   $ ballerina build flight_booking_service --skiptests
```

Comment:
The `--skiptests` flag should come before the package name. i.e., `ballerina build --skiptests flight_booking_service`